python/lsst/obs/base/fitsRawFormatterBase.py

```diff
@@ -225,3 +240,12 @@ def observationInfo(self):
         if self._observationInfo is None:
             self._observationInfo = ObservationInfo(self.metadata, translator_class=self.translatorClass)
         return self._observationInfo
+
+    @property
+    def metadata(self):
```

Comment:
This is overriding the base class but the base class implementation is the same. Please remove.

Reply:
Thanks, I'd missed that during my rebasing.